packages/react/src/SubdomainNavBar/SubdomainNavBar.tsx

```diff
@@ -16,6 +16,7 @@ import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/compone
 
 /** * Main Stylesheet (as a CSS Module) */
 import styles from './SubdomainNavBar.module.css'
+import {useId} from '@reach/auto-id'
```

Comment:
Nit: We have our own internal `useId` hook btw, which might be worth using here. Right now takes a dependency on `'@reach/auto-id'`, but helps centralise usage of it so we can find and replace them more easily in future.

Reply:
Everywhere else we are using `useId` it imports from `@react/auto-id`